Mathmagician.Tests/EvenNumberTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class EvenNumberTests
+    {
+        [TestMethod]
+        public void MakeSureYouCanMakeEvenNumber()
+        {
+            //Arrange
+
+            //Act
+            EvenNumber usereven = new EvenNumber();
+
+            //Assert
+            Assert.IsNotNull(usereven);
+        }
+
+
+        [TestMethod]
+        public void CheckIfUserNumberIsEven()
+        {
+            EvenNumber neweven = new EvenNumber();
+
+            neweven.usereven = 7;
+
+            Assert.IsFalse(neweven.usereven % 2 == 0);
+
+        }
```

Comment:
@jcockhren I think it's showing the wrong code again on this pull request because that is not the code that I have for the Evens.cs or the EvenNumberTests.cs files